app/views/common/_category_nav.html.erb

```diff
@@ -1,6 +1,7 @@
-<ul class="nav nav-stacked">
+<ul class="nav nav-pills nav-stacked">
   <% @categories.each do |category| %>
-    <li>
+    <% css_class = @category == category ? "active" : "" %>
```

Comment:
We should probably start pulling logic like this to helper methods so we can keep logic out of the view. Or if you want to leave it here, can we name ```css_class``` something that better reveals we're setting the ```active``` class?

Or just move it inline, since it's short: 

```<li class="@category == category ? "active" : ""%>">